infer/environment.cc

```diff
@@ -1189,7 +1190,7 @@ core::TypePtr Environment::processBinding(core::Context ctx, cfg::Binding &bind,
         }
         throw;
     }
-}
+} // namespace sorbet::infer
```

Comment:
Was this accidentally added? You might have temporarily removed a brace, saved, linting added this comment, and later fixed the braces.

Reply:
Ah, that's almost certainly what happened.  Will remove.

Reply:
Removed!